tasks-by-users/AbbasShaikh0304/01-E-Commerce website/products.html

```diff
@@ -0,0 +1,272 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/fontawesome.min.css"
+    />
+
+    <link rel="stylesheet" href="Style.css" />
+    <title>All Products</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="navbar">
+        <div class="logo">
+          <a href="index.html"><img src="images/logo.png" alt="RedStore" width="125px" /></a>
+        </div>
+        <nav>
+          <ul id="MenuItems">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="products.html">Products</a></li>
+            <li><a href="">About</a></li>
+            <li><a href="">Contacts</a></li>
+            <li><a href="account.html">Account</a></li>
+          </ul>
+        </nav>
+        <a href="cart.html"><img src="images/cart.png" alt="Cart" width="30px" height="30px" /></a>
+        <img src="images/menu.png" class="menu-icon" onclick="menutoggle()" />
+      </div>
+    </div>
+    <div class="small-container">
+      <div class="row row-2">
+        <h2>All Products</h2>
+        <select>
```

Comment:
do you know the difference between ` <select> ` and ` <datalist> ` tags??
@AbbasShaikh0304

Reply:
Yes. Select input element presents options for the users from which they need to select one of them. On the otherhand, Datalist presents a list of suggested and users are free to select one of those suggested values or type in their own value. @Tauqeer-Ahmed-99